component/component/vshn_postgres.jsonnet

```diff
@@ -856,12 +197,16 @@ local composition(restore=false) =
                       sgNamespace: pgParams.sgNamespace,
                       externalDatabaseConnectionsEnabled: std.toString(params.services.vshn.externalDatabaseConnectionsEnabled),
                       quotasEnabled: std.toString(params.services.vshn.quotasEnabled),
+                      plans: std.toString(pgPlans),
```

Comment:
In light of my comment in the other PR: it might make sense to introduce argoCD ordering for the component.

my suggestion for the ordering:
* compositions
* providers
* comp functions

That way we can ensure that argocd deploys the composition and newly specified inputs first.